### ISSUE TITLE
Accessibility Test || Web App || Issues noted on the Course page 

### ISSUE THREAD

**Issue by poojaustad** [UstadMobile/UstadMobile#370]
**Describe the bug**
Issues noted on the course page 

1. Course start date is not visible to a user but the narrator able to narrate it
2.  Expand - collapse buttons are narrated as buttons- not specified its functionality
3. To open a discussion board there is a button available - it's not visible to the user but narrated by the narrator
4. Users not able to open a text/assignment/content block by clicking on the ENTER key 

**To Reproduce**

1. Go to Courses- open the course
2. Course start date is not visible to a user but the narrator able to narrate it
3. Expand - collapse buttons are narrated as buttons- not specified its functionality
4. To open a discussion board there is a button available - it's not visible to the user but narrated by the narrator
5. Users not able to open a text/assignment/content block by clicking on the ENTER key 

**Expected behavior**
1. Course start date should be visible to a user 
2.  Expand - collapse buttons should be narrated specifying their functionality
3. To open a discussion board there is a button  visible to the user that's narrated by the narrator
4. Users should be able to open a text/assignment/content block by clicking on the ENTER key 

**Screenshots**
![2022-07-15_10h36_56](https://user-images.githubusercontent.com/78338755/179166386-10cda6c9-08fb-428c-8c1c-9eb842872255.png)

![2022-07-15_10h37_40](https://user-images.githubusercontent.com/78338755/179166408-9dd88796-6574-438b-a0e9-71a3127b339d.png)

![2022-07-15_10h38_56](https://user-images.githubusercontent.com/78338755/179166429-0a184b79-153c-4b4c-8d6d-ae5fc18764e6.png)
